src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -538,6 +540,22 @@ public Locomotive switchToDefaultContent() {
     /* ************************ */
 
     /* Validation Functions for Testing */
+    public Locomotive softAssert(String css) {
```

Comment:
The name should be more descriptive?

Reply:
One of these maybe?
softAssertIsPresent()
softAssertTrueIsPresent()
softValidatePresent()
softAssertValidatePresent()